marimo/_plugins/ui/_impl/input.py

```diff
@@ -914,17 +914,6 @@ def __init__(
         full_width: bool = False,
         max_selections: Optional[int] = None,
     ) -> None:
-        if len(options) > multiselect._MAX_OPTIONS:
```

Comment:
can you try a _really_ large list (e.g. 10,000)? maybe we still need this, but can just bump it higher. 

I saw you commited a notebook - https://github.com/marimo-team/marimo/pull/1654/commits/5471d9f0b96f9cf1f8fd975bae48bdf7c3d96ac5
we actually like doing this - and putting them in `mairmo/_smoke_tests/bugs/<issue_number_or_name>.py`

Reply:
Oh cool! Yeah let me test with small / large lists and add it back then.

Reply:
Seems fine! Tested with 10,000 items and 100,000 items. However, with 1M items:
```
Your output is too large for marimo to show. It has a size of 23889316 bytes. Did you output this object by accident?
```
I'll add this limit and its test back in.